kt/kt.go

```diff
@@ -313,57 +313,57 @@ func (c *Conn) doRPC(path string, values []kv) (code int, vals []kv, err error)
 	if err != nil {
 		return 0, nil, err
 	}
-	m := decodeValues(resultBody, resp.Header.Get("Content-Type"))
+	m := DecodeValues(resultBody, resp.Header.Get("Content-Type"))
 	return resp.StatusCode, m, nil
 }
 
 type encoding int
```

Comment:
Leaving this unexported means you can't define a variable to hold `IdentityEnc`/`Base64Enc` without a `:=` trick, or a function that takes them. Might be acceptable, but FYI.